core/src/main/java/greencity/controller/UserController.java

```diff
@@ -1125,4 +1147,24 @@ public ResponseEntity<HttpStatus> deactivateEmployee(@RequestParam String uuid)
         userService.markUserAsDeactivated(uuid);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Controller that activate employee by uuid.
+     *
+     * @param uuid - uuid of Employee.
+     * @author Oksana Spodaryk.
+     */
+    @ApiOperation(value = "Activate employee by uuid")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @PutMapping("/activate-employee")
```

Comment:
it seems like there are two endpoints (`/activate-employee` and `/markUserAsActivated`) with the same functionality
Why do we need this code duplication?

Reply:
deleted duplicates